cmd/terramate/cli/cli.go

```diff
@@ -812,10 +812,6 @@ func (c *cli) format() {
 }
 
 func (c *cli) printStacks() {
-	if c.parsedArgs.Changed {
```

Comment:
@mariux I'm not sure if you want this removed now or later. Please let me know.

Reply:
yes, can be dropped.